### PR TITLE
Setup/mutation

### LINKED ISSUE #18: User can add new prescription information and POST (Mutation) to add to their account

### PR DESCRIPTION
**Pull request by artanmyrtolli**
### Description

Finalized mutation that fires when user submits form of adding new Rx. 

Fixes #18 

### Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)
- [x ] This change requires a documentation update

### Checklist:

- [ x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/App/App.tsx [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import React, { useEffect, useState } from 'react';
-// import { Route } from 
 import './App.css';
 import Nav from '../../Components/Nav/Nav'
 import { User } from '../../interfaces'
-import { user1 } from '../../mockData'
-import { dummyUser } from '../../interfaces'
-import { dummyMed } from '../../interfaces'
+import { QueryRx } from '../../interfaces'
```
Comment (hemeshvpatelHE) [inline src/Components/App/App.tsx:5]: Should be able to import multiple interfaces if from the same file:

`import { User, QueryRx } from '../../interfaces'`

**Review by angelewilliams — COMMENTED** [review]
Looks good, great job getting the mutation to work!

File: src/Components/Dashboard/MedReminder.tsx [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
 type MedProps = {
-    med: dummyMed
+    med: QueryRx
 }
 
 const MedReminder: React.FC<MedProps> = ({ med }) => {
     const [infoHover, setInfoHover] = useState('hidden')
-
+    const [refillHover, setRefillHover] = useState('hidden')
```
Comment (hemeshvpatelHE) [inline src/Components/Dashboard/MedReminder.tsx:13]: Good opportunity to set types on useState (infoHover and refillHover)

File: src/Components/Form/SearchForm.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ type UserProps = {
 }
 
 const SearchForm: React.FC<UserProps> = ({ userID }) => {
+  console.log(userID)
```
Comment (hemeshvpatelHE) [inline src/Components/Form/SearchForm.tsx:10]: Don't forget to remove unless using for testing

File: src/Components/Form/SubmissionForm.tsx [hunk 4]
```diff
@@ -79,17 +102,35 @@ const SubmissionForm: React.FC<MedProps> = ({ chosenMedicine }) => {
 
             </div>
             <div className="dosage-section">
-                <label htmlFor="dosage-num">Dosage: </label>
+                <label htmlFor="dosage-num">Single Dose :</label>
```
Comment (hemeshvpatelHE) [inline src/Components/Form/SubmissionForm.tsx:105]: Checked a few other labels, would recommend keeping ':' in same placement throughout. 

`Single Dose:`

**Review by emmacbd — APPROVED** [review]
Thank for all the tinkering with mutations, looks good.

**Review by shanekwarning — APPROVED** [review]
Great work communicating with backend and being able to set up the mutation with GraphQL.

**Pull request merged**